.gitattributes

```diff
@@ -0,0 +1 @@
+esm/*.js binary
```

Comment:
Hmm, this doesn't appear to have worked for this diff in the mobile app. I think this is ok, but I would've used the `diff` modifier, I think (that's what I used in render-gateway).

Reply:
In `webapp` we use `binary`, as far as I can tell.